include/CLI/Validators.hpp

```diff
@@ -467,35 +467,35 @@ class Range : public Validator {
     /// Note that the constructor is templated, but the struct is not, so C++17 is not
     /// needed to provide nice syntax for Range(a,b).
     template <typename T>
-    Range(T min, T max, const std::string &validator_name = std::string{}) : Validator(validator_name) {
+    Range(T minVal, T maxVal, const std::string &validator_name = std::string{}) : Validator(validator_name) {
```

Comment:
```suggestion
    Range(T min_val, T max_val, const std::string &validator_name = std::string{}) : Validator(validator_name) {
```
(And for the others). I think this is more consistent.

Reply:
made the change